grc: tests on test_qtbot fail

### What happened?

test_qtbot fail due to changes on grc-qt GUI

This issue is due to the changes introduced in https://github.com/gnuradio/gnuradio/pull/7435

Btw I tried to fix this with simulating Key_Down and Key_Enter (modifying `add_block_from_query` in `test_qtbot.py`) but it does not work.
code:
```
def add_block_from_query(qtbot, app, query):
    qtbot.keyClick(app.focusWidget(), QtCore.Qt.Key_F, QtCore.Qt.ControlModifier)
    type_text(qtbot, app, query)
    keystroke(qtbot, app, QtCore.Qt.Key_Down)
    keystroke(qtbot, app, QtCore.Qt.Key_Enter)
```
test result:
![image](https://github.com/user-attachments/assets/3d20b012-e276-4253-a555-43c0c48243dc)


### System Information

OS:
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.4 LTS
Release:	22.04
Codename:	jammy

GR Installation Method: from source


### GNU Radio Version

3.11-git (main)

### Specific Version

v3.11.0.0git-799-g2a8b58ca

### Steps to Reproduce the Problem

run `pytest grc/tests`

### Relevant log output

```shell
FAILED tests/test_qtbot.py::test_add_null_sink - assert None is not None
FAILED tests/test_qtbot.py::test_add_null_source - assert None is not None
FAILED tests/test_qtbot.py::test_add_throttle - assert None is not None
FAILED tests/test_qtbot.py::test_right_click - assert None is not None
FAILED tests/test_qtbot.py::test_errors - assert None is not None
FAILED tests/test_qtbot.py::test_move_blocks - assert None is not None
FAILED tests/test_qtbot.py::test_connection - AttributeError: 'NoneType' object has no attribute 'gui'
FAILED tests/test_qtbot.py::test_num_inputs - AttributeError: 'NoneType' object has no attribute 'sinks'
FAILED tests/test_qtbot.py::test_bus - AttributeError: 'NoneType' object has no attribute 'sinks'
FAILED tests/test_qtbot.py::test_bypass - AttributeError: 'NoneType' object has no attribute 'gui'
FAILED tests/test_qtbot.py::test_generate - AttributeError: 'NoneType' object has no attribute 'gui'
ERROR tests/test_qtbot.py::test_errors - Failed: TEARDOWN ERROR: Exceptions caught in Qt event loop:
```

--- a/grc/tests/test_qtbot.py
+++ b/grc/tests/test_qtbot.py
@@ -98,7 +98,11 @@ def gather_menu_items(menu):
 def add_block_from_query(qtbot, app, query):
     qtbot.keyClick(app.focusWidget(), QtCore.Qt.Key_F, QtCore.Qt.ControlModifier)
     type_text(qtbot, app, query)
-    keystroke(qtbot, app, QtCore.Qt.Key_Enter)
+    qtbot.wait(10)
+    pag.press('down')
+    qtbot.wait(10)
+    pag.press('enter')
+    qtbot.wait(10)
 
 
 def find_blocks(flowgraph, block_type):
@@ -186,7 +190,6 @@ def test_delete_block(qtbot, qapp_cls_):
 def test_add_null_sink(qtbot, qapp_cls_):
     qtbot.wait(100)
     add_block_from_query(qtbot, qapp_cls_, "null sin")
-    qtbot.wait(100)
 
     n_sink = find_blocks(qapp_cls_.MainWindow.currentFlowgraph, "blocks_null_sink")
     assert n_sink is not None

